packages/tests/sample.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: appcat.vshn.io/v1
+kind: ExoscaleOpensearch
+metadata:
+  name: test-opensearch-44
+spec:
+  writeConnectionSecretToRef:
+    name: "hello-world-opensearch-44"
+#  parameters:
+#    network:
+#      ipFilter:
+#        - 0.0.0.0/0
+#    backup:
+#      timeOfDay: "12:00:00"
+#    maintenance:
+#      timeOfDay: "12:10:00"
+#      dayOfWeek: "tuesday"
+#    service:
+#      version: "2"
+#      zone: "ch-gva-2"
+#    size:
+#      plan: "startup-4"
```

Comment:
Could you remove that. We don't have any samples for the other DBaaS and it definitely shouldn't be in the test directory.

Reply:
sure

Reply:
Thanks, but please also update the make target as it tries to apply a manifest that doesn't exist anymore